setup.py

```diff
@@ -1,106 +0,0 @@
-#!/usr/bin/env python
-
-import os
-import os.path
-from setuptools import setup
-
-NAME = 'iris_grib'
-PYPI_NAME = 'iris-grib'
-PACKAGE_DIR = os.path.abspath(os.path.dirname(__file__))
-PACKAGE_ROOT = os.path.join(PACKAGE_DIR, NAME)
-
-packages = []
-for d, _, _ in os.walk(os.path.join(PACKAGE_DIR, NAME)):
-    if os.path.exists(os.path.join(d, '__init__.py')):
-        packages.append(d[len(PACKAGE_DIR) + 1:].replace(os.path.sep, '.'))
-
-
-def pip_requirements(*args):
-    requirements = []
-    for name in args:
-        fname = os.path.join(
-            PACKAGE_DIR, "requirements", "{}.txt".format(name)
-        )
-        if not os.path.exists(fname):
-            emsg = (
-                f"Unable to find the {name!r} requirements file at {fname!r}"
-            )
-            raise RuntimeError(emsg)
-        with open(fname, "r") as fh:
-            for line in fh:
-                line = line.strip()
-                if not line or line.startswith("#"):
-                    continue
-                requirements.append(line)
-    return requirements
-
-
-def extract_version():
-    version = None
-    fname = os.path.join(PACKAGE_DIR, 'iris_grib', '__init__.py')
-    with open(fname) as fi:
-        for line in fi:
-            if (line.startswith('__version__')):
-                _, version = line.split('=')
-                version = version.strip()[1:-1]  # Remove quotations
-                break
-    return version
-
-
-def long_description():
-    fname = os.path.join(PACKAGE_DIR, "README.rst")
-    with open(fname, "rb") as fi:
-        result = fi.read().decode("utf-8")
-    return result
-
-
-def file_walk_relative(top, remove=''):
-    """
-    Returns a generator of files from the top of the tree, removing
-    the given prefix from the root/file result.
-
-    """
-    top = top.replace('/', os.path.sep)
-    remove = remove.replace('/', os.path.sep)
-    for root, dirs, files in os.walk(top):
-        for file in files:
-            yield os.path.join(root, file).replace(remove, '')
-
-
-setup_args = dict(
-    name             = PYPI_NAME,                                # noqa: E221,E251,E501
-    version          = extract_version(),                        # noqa: E221,E251,E501
-    packages         = packages,                                 # noqa: E221,E251,E501
-    package_data     = {'iris_grib': list(                       # noqa: E221,E251,E501
-        file_walk_relative('iris_grib/tests/results', remove='iris_grib/'))
```

Comment:
We will need to create a `MANIFEST.in` file to replicate this one - [docs](https://setuptools.pypa.io/en/stable/userguide/datafiles.html#include-package-data).

Looks like no changes would be needed in `pyproject.toml` itself, since it's all on by default.

Reply:
OK I ran `python -m build` and compared the contents of `iris_grib/tests/results` to the original package. Everything is there. I'm afraid I don't know why but happy to trust it.